### ISSUE TITLE
Possible Classes

### ISSUE THREAD

**Issue by ClickerMonkey** [ClickerMonkey/unitz-ts#1]
- USD
- Energy
- Frequency
- Weight -> Mass
- Pressure

**Comment by nealot** [comment]
Is it possible to convert between bases? @ClickerMonkey

**Comment by ClickerMonkey** [comment]
Yes, as seen in here : https://github.com/ClickerMonkey/unitz-ts/blob/master/src/classes/Length.ts
